tests/unit/test_deprecations.py

```diff
@@ -34,3 +34,8 @@ def test_subreddit_rules_call(self):
             excinfo.value.args[0] == "Calling SubredditRules to get a list of "
             "rules is deprecated. Remove the parentheses to use the iterator."
         )
+
+    def test_reddit_user_me_read_only(self):
+        self.reddit.read_only = True
+        with pytest.raises(DeprecationWarning):
+            self.reddit.user.me()
```

Comment:
Check the message as well.
```suggestion
        with pytest.raises(DeprecationWarning) as excinfo:
            self.reddit.user.me()
        assert excinfo.value.args[0] == "The None return value is deprecated. Consider "
                "testing Reddit.read_only before calling this method. "
                "From PRAW 8 onwards this method will start raising an "
                "error when called in read-only mode."
```

This snippet should be Black-ified though.